Source/ACE.Server/WorldObjects/Player.cs

```diff
@@ -69,6 +69,8 @@ public Player(Weenie weenie, ObjectGuid guid, uint accountId) : base(weenie, gui
             Attackable = true;
 
             SetProperty(PropertyString.DateOfBirth, $"{DateTime.UtcNow:dd MMMM yyyy}");
+
+            SetProperty(PropertyString.GodState, $"0={DateTime.UtcNow}");
```

Comment:
I'm not sure this is needed.

When you ungod, you should RemoveProperty(). You don't need to set a default property here. The absence of the property should mean that there is no UnGod state

Reply:
I was thinking it could be useful to have the timestamps for both directions in case of the need to throttle the requests, or debug in case of errors.

RemoveProperty() would remove the entry from the DB completely? So I would need to check for null, or just that the entry does not exist, instead of StartsWith("0")?

Removing the property from the DB completely when not in godmode would help keep things tidy.

Reply:
Correct, you would just check if the property was null. A null property is one that does not exist.

Reply:
cleaned it up and changed it to remove the property instead of set to "0=xxxx"